speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/frontend/ThresholdConfig.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "ConfigEnums.h"
+#include "../simulatedDrivingAssistance/rppUtils/RppUtils.hpp"
+#include "../simulatedDrivingAssistance/rppUtils/FileDialog.hpp"
+
+#define PARAM_FILE    "config/Threshhold.xml"
```

Comment:
Clarify name for usage outside of this class, PARAM_FILE could be anything.
```suggestion
#define THRESHOLD_CONFIG_FILE    "config/Threshhold.xml"
```